test/Integration/tpp-run-splat-mlp.mlir

```diff
@@ -53,7 +53,7 @@ func.func @entry(%arg0: tensor<8x8xf32>, %output: tensor<8x8xf32>) -> tensor<8x8
 // CHECK-DAG: memref.global "private" constant @__constant_1x1x8x8xf32 : memref<1x1x8x8xf32>
 // CHECK-DAG: memref.global "private" constant @__constant_1x1x8x8xf32_0 : memref<1x1x8x8xf32>
 // CHECK-DAG: memref.global "private" constant @__constant_8xf32 : memref<8xf32>
-// CHECK-DAG: memref.global "private" constant @__constant_8xf32_0 : memref<8xf32>
+// CHECK-DAG: memref.global "private" constant @__constant_8xf32_1 : memref<8xf32>
```

Comment:
Any clue where this change is coming from or if there's any actual change other than name?
My guess is the bufferization state update caused this.

Reply:
No change besides the name. Other constants which share a type still have `_0` as the first repeated occurrence of a constant of that type. There's no `@__constant_8xf32_0` at all in the IR any more. My guess would be that some change upstream duplicated `@__constant_8xf32_0` so that there were `@__constant_8xf32_0` and `@__constant_8xf32_1` and some canonicalization/cse got rid of the duplicate constant. Just a guess though.